docs/blockbuilder.md

```diff
@@ -158,29 +199,30 @@ class QuoteBlock(blocks.StructBlock):
         template = "wagtail_wordpress_import/quote_block.html"
 ```
 
-## Included Fallback/Catch-all Block
+### Included Fallback Block
 
 By default, the fallback block is a Wagtail `RichText` Block.
 
 Only content that has no specific block filter is added to the fallback block.
 
 Example: `<p> <ul> <a> <img /> ...`
 
-This block is only saved to the block sequence each time the builder finds a new Block is required or the builder has reached the end of the content parsing.
+*This block is only saved to the block sequence each time the builder finds a new Block is required in the sequence or the builder has reached the end of the content parsing.*
```

Comment:
Say we have the following HTML thats been processed by the block builder.

```html
<h1>Heading</h1> <!-- heading block --> 
<p>Some content</p> <!-- rich text block 1 --> (cached)
<p>Some content</p> <!-- rich text block 1 --> (cached)
<p>Some content</p> <!-- rich text block 1 --> (cached)
<blockquote><p>A quote</p></blockquote> <!-- block quote block -->
<h2>heading</h2> <!-- rich text block 2 -->  (cached)
<p>Some content</p> <!-- rich text block 2 --> (cached)
<p>Some content</p> <!-- rich text block 2 --> (cached)
<table>...</table> <!-- raw html block -->
<p>Some content</p> <!-- rich text block 3 --> (cached)
<p>Some content</p> <!-- rich text block 3 --> (not cached because it's the last tag and the sequence is completed)
```
We end up with 6 block types.